scalafix/input/src/main/scala/incompat/ScalafixIncompat11.scala

```diff
@@ -14,5 +14,5 @@ object Incompat11 {
   implicit class RichFuture[T](future: Future[T]) {
     def await(implicit duration: Duration = 10.seconds): T = Await.result[T](future, duration)
   }
-  def test = Future(new Foo()).await.status
+  def test = Future.apply[incompat.Incompat11.Foo](new Foo()).await.status
```

Comment:
This was maybe added by mistake?

Reply:
yes ^^ during the investigation, to reduce the number of synthetics.